.github/workflows/wheels.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: Wheel Builds
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  build_wheels_aarch64:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==2.0.1 twine
+      - name: Build wheels
+        run: |
+          python -m cibuildwheel --output-dir wheelhouse
+        env:
+          CIBW_SKIP: cp27-* cp34-* cp35-* pp*
```

Comment:
This is mostly a legacy thing, we've been using cibw since before `CIBW_BUILD` was an option. This warning is just because I have an explict skip listed for python 2.7, 3.4, and 3.5 none of which cibw>=2 supports. So this is just cibw warning that we don't need these skips anymore. I probably should just remove those skips from this build config

Reply:
Ah yeah, that makes sense - I never considered it might be because skip was overspecified.  Perhaps let's leave it for now (it's not hurting) in order to land the PR and get 0.18.1 today, and we can fix build warnings in another PR for 0.19.